### PR TITLE
Agrego material relacionado con crear un repo y crear el perfil

### PR DESCRIPTION
**Pull request by yabellini**
- [x] GitHub profile
- [x] Repositories
- [ ] GitHub Issues
- [ ] Release and integration with other apps (Zenodo)

### CODE REVIEW

**Comment by yabellini** [comment]
@beatrizmilz la leccion de perfil y repositorios esta lista.  Por si querés revisarlos.  
La de issues todavia tengo que terminarla.

File: profile.qmd [hunk 1]
```diff
@@ -1,3 +1,75 @@
...
+Si revisamos [el perfil en GitHub de Yanina](https://github.com/yabellini) vamos a ver un par de cosas interesantes:
+
+- _Informacion general:_  llamamos a esto todo lo que aparece debajo de la foto del perfil.  Nombre, pronombres y una mini bio.  Luego datos de cuantos seguidores tienen y a cuantas personas sigue. Ademas se puede detallar:
+
+  * Lugar de trabajo: @ropensci
+  * Localizacion: Santa Rosa, La Pampa, Argentina
+  * Zona horaria: 12:51 (UTC -03:00)
+  * Sitio web: https://yabellini.netlify.app
+  * ORCID: https://orcid.org/0000-0002-4522-7466
+  * Diversas redes sociales como Mastodon, LinkedIn, BlueSky y Youtube.
```
Comment (beatrizmilz) [inline profile.qmd:35]:
```suggestion
  * Lugar de trabajo: [@ropensci](https://github.com/ropensci)
  * Localizacion: Santa Rosa, La Pampa, Argentina
  * Zona horaria: 12:51 (UTC -03:00)
  * Sitio web: <https://yabellini.netlify.app>
  * ORCID: <https://orcid.org/0000-0002-4522-7466>
  * Diversas redes sociales como Mastodon, LinkedIn, BlueSky y Youtube. 
```

File: profile.qmd [hunk 2]
```diff
@@ -1,3 +1,75 @@
...
+- Uno de los pasos implica chequear que son una persona y luego validar la direccion de email que usaste para crear la cuenta. 
+
+
+## Nuestro perfil
+
+Una vez que creamos una cuenta en GitHub, tenemos nuestro perfil publico en internet. 
+
+La página de tu perfil cuenta la historia de tu trabajo a través de los repositorios en los que te interesas, las colaboraciones que has realizado y las conversaciones que has tenido.
+
+Si revisamos [el perfil en GitHub de Yanina](https://github.com/yabellini) vamos a ver un par de cosas interesantes:
```
Comment (beatrizmilz) [inline profile.qmd:26]:
```suggestion
Si revisamos [el perfil en GitHub de Yanina](https://github.com/yabellini) vamos a ver un par de cosas interesantes  en columna izquierda de la página:
```

File: profile.qmd [hunk 3]
```diff
@@ -1,3 +1,75 @@
...
+- _Informacion general:_  llamamos a esto todo lo que aparece debajo de la foto del perfil.  Nombre, pronombres y una mini bio.  Luego datos de cuantos seguidores tienen y a cuantas personas sigue. Ademas se puede detallar:
+
+  * Lugar de trabajo: @ropensci
+  * Localizacion: Santa Rosa, La Pampa, Argentina
+  * Zona horaria: 12:51 (UTC -03:00)
+  * Sitio web: https://yabellini.netlify.app
+  * ORCID: https://orcid.org/0000-0002-4522-7466
+  * Diversas redes sociales como Mastodon, LinkedIn, BlueSky y Youtube. 
+
+- _Notificaciones y logros_:destacan tu actividad y se muestran si utilizas GitHub Pro o si participas en programas como Arctic Code Vault, GitHub Sponsors o GitHub Stars.
```
Comment (beatrizmilz) [inline profile.qmd:37]:
```suggestion
- _Notificaciones y logros_: destacan tu actividad y se muestran si utilizas GitHub Pro o si participas en programas como Arctic Code Vault, GitHub Sponsors o GitHub Stars.
```

File: profile.qmd [hunk 4]
```diff
@@ -1,3 +1,75 @@
...
+
+- _Organizaciones_: un listado de las organizaciones que estan presentes en GitHub y de las cuales sos parte.  Esto se completa automaticamente, aunque puedes decidir si algunas cosas son visibles o no.
+
+::: ejercicio
+
+Entra a tu perfil y completa con tus datos relevantes, como por ejemplo una bio, tus pronombres, tus redes sociales, donde estas y tu zona horaria.
```
Comment (beatrizmilz) [inline profile.qmd:43]:
```suggestion
Entra a tu perfil, clique en "Edit Profile", y completa con tus datos relevantes, como por ejemplo una bio, tus pronombres, tus redes sociales, donde estas y tu zona horaria.
```

**Comment by beatrizmilz** [comment]
Review Bea:
- [ ] GitHub profile
- [ ] Repositories
- [ ] GitHub Issues
- [ ] Release and integration with other apps (Zenodo)

File: profile.qmd [hunk 5]
```diff
@@ -1,3 +1,75 @@
...
+
+::: 
+
+## Informacion de nuestra actividad
+
+Los perfiles ademas muestran nuestros repositorios y nuestra actividad mas reciente. 
+
+Tambien podemos personalizarlo agregando un resumen de nuestra actividad. 
+
+Podemos seleccionar que mostrar de cada una de estas secciones.  Por ejemplo podemos fijar hasta seis repositorios en nuestro perfil y seran los primeros que las personas vean cuando entren a nuestra pagina.
```
Comment (beatrizmilz) [inline profile.qmd:55]:
```suggestion
Podemos seleccionar que mostrar de cada una de estas secciones.  Por ejemplo podemos fijar hasta seis repositorios en nuestro perfil y seran los primeros que las personas vean cuando entren a nuestra pagina.

En la imagene abajo, tenemos los repositorios fijados en lo perfil de Bea:

![Captura de pantalla - Perfil de Bea - Repositorios fijados](images/profile-pin-bea.png)
```

File: profile.qmd [hunk 6]
```diff
@@ -1,3 +1,75 @@
...
+  * Localizacion: Santa Rosa, La Pampa, Argentina
+  * Zona horaria: 12:51 (UTC -03:00)
+  * Sitio web: https://yabellini.netlify.app
+  * ORCID: https://orcid.org/0000-0002-4522-7466
+  * Diversas redes sociales como Mastodon, LinkedIn, BlueSky y Youtube. 
+
+- _Notificaciones y logros_:destacan tu actividad y se muestran si utilizas GitHub Pro o si participas en programas como Arctic Code Vault, GitHub Sponsors o GitHub Stars.
+
+- _Organizaciones_: un listado de las organizaciones que estan presentes en GitHub y de las cuales sos parte.  Esto se completa automaticamente, aunque puedes decidir si algunas cosas son visibles o no.
+
```
Comment (beatrizmilz) [inline profile.qmd:40]:
```suggestion

![Captura de pantalla - Perfil de Yani](images/profile-yani.png)
```

File: profile.qmd [hunk 7]
```diff
@@ -1,3 +1,75 @@
...
+Entra a tu perfil y completa con tus datos relevantes, como por ejemplo una bio, tus pronombres, tus redes sociales, donde estas y tu zona horaria.
+
+Cuando termines comparti el link a tu perfil en el chat y en el documento compartido. 
+
+::: 
+
+## Informacion de nuestra actividad
+
+Los perfiles ademas muestran nuestros repositorios y nuestra actividad mas reciente.
```
Comment (beatrizmilz) [inline profile.qmd:51]:
```suggestion
Los perfiles ademas muestran nuestros repositorios y nuestra actividad mas reciente. 
En la imagen de abajo, tenemos las actividades más recientes en el [perfil de GitHub de Yani](https://github.com/yabellini):
```

File: profile.qmd [hunk 8]
```diff
@@ -1,3 +1,75 @@
...
+
+Cuando termines comparti el link a tu perfil en el chat y en el documento compartido. 
+
+::: 
+
+## Informacion de nuestra actividad
+
+Los perfiles ademas muestran nuestros repositorios y nuestra actividad mas reciente. 
+
+Tambien podemos personalizarlo agregando un resumen de nuestra actividad.
```
Comment (beatrizmilz) [inline profile.qmd:53]:
```suggestion
Tambien podemos personalizarlo agregando un resumen de nuestra actividad. En la imagen de abajo, tenemos el resumen del [perfil de GitHub de Yani](https://github.com/yabellini):
![Captura de pantalla - Perfil de Yani - Resumen](images/profile-yani-readme.png)
```

**Comment by yabellini** [comment]
Yes! What is better for you.

**Comment by yabellini** [comment]
Thanks Bea for all the great changes and additions!

**Pull request merged**